core/shared/src/main/scala/zio/concurrent/ConcurrentSet.scala

```diff
@@ -0,0 +1,42 @@
+package zio.concurrent
+
+import zio.UIO
+
+import java.util.concurrent.ConcurrentHashMap
+import scala.jdk.CollectionConverters._
+
+final class ConcurrentSet[A] private (underlying: ConcurrentHashMap.KeySetView[A, java.lang.Boolean]) {
```

Comment:
Do the data structures exist in scala-js and native?

Reply:
Yes. https://github.com/scala-js/scala-js/blob/master/javalib/src/main/scala/java/util/concurrent/ConcurrentHashMap.scala